config/core.yml

```diff
@@ -1,16 +1,17 @@
 base: &default
   url: "<%= File.join(ENV.fetch('BACKEND_URL', 'http://localhost:3001'), '/internal/') %>"
-  # fake server is useful when you don't run a real backend apisonator
-  # fake_server: 'http://localhost:3000/internal/'
-#  fake_server: false
+  fake_server: false
   username: <%= ENV.fetch('CONFIG_INTERNAL_API_USER', nil) %>
   password: <%= ENV.fetch('CONFIG_INTERNAL_API_PASSWORD', nil) %>
 
 development:
   <<: *default
+  # fake server is useful when you don't run a real backend apisonator
+  # fake_server: 'http://localhost:3000/internal/'
```

Comment:
OK, this might be a potential place that a developer might want to change locally. We can add an env var for this.